docs/documentation/platform/gateways/relay-deployment.mdx

```diff
@@ -0,0 +1,424 @@
+    ```
+
+    **Step 2: Relay Deployment**
+
+    Deploy the relay using the **same secret value**:
+
+    ```bash
+    # Set the matching secret for relay deployment
+    export INFISICAL_RELAY_AUTH_SECRET=<shared-secret-value>
+
+    # Start the instance relay
+    infisical relay start --type=instance --host=<public-ip> --name=<relay-name>
+    ```
+
+    **Install as daemon (Linux only):**
+
+    ```bash
+    # Instance relay
+    sudo infisical relay systemd install --type=instance --name=<name> --host=<host> --relay-auth-secret=<secret>
+
+    # Start the service
+    sudo systemctl start infisical-relay
+
+    # Uninstall
+    sudo infisical relay systemd uninstall
+    ```
+
+    <Warning>
+      The `RELAY_AUTH_SECRET` (instance environment) and
+      `INFISICAL_RELAY_AUTH_SECRET` (relay environment) **must match exactly** for
+      authentication to work.
+    </Warning>
+
+  </Accordion>
+
+  <Accordion title="Organization Relays">
+    Deploy your own relay server for full control. Organization relays support all authentication methods:
+
+    ```bash
+    # Token auth
+    infisical relay start --type=org --host=<public-ip> --name=<relay-name> --token=<token>
+
+    # Universal auth
+    infisical relay start --type=org --host=<public-ip> --name=<relay-name> --auth-method=universal-auth --client-id=<id> --client-secret=<secret>
+
+    # Other auth methods (kubernetes, aws-iam, gcp-id-token, azure, etc.)
+    infisical relay start --type=org --host=<public-ip> --name=<relay-name> --auth-method=<method> --machine-identity-id=<id>
+    ```
+
+    **Install as daemon (Linux only):**
+
+    ```bash
+    # Organization relay
+    sudo infisical relay systemd install --token=<token> --name=<name> --host=<host> --type=org
+
+    # Start the service
+    sudo systemctl start infisical-relay
+
+    # Uninstall
+    sudo infisical relay systemd uninstall
+    ```
+
+  </Accordion>
+</AccordionGroup>
+
+## Network Configuration
+
+### Relay Server Network Requirements
+
+The relay server must accept inbound connections and make outbound connections to function properly:
+
+#### Inbound Connections (Required)
+
+The relay server must accept the following inbound connections:
+
+| Protocol | Source             | Port | Purpose                          |
+| -------- | ------------------ | ---- | -------------------------------- |
+| TCP      | Gateways           | 2222 | SSH reverse tunnel establishment |
+| TCP      | Infisical Platform | 8443 | Platform-to-relay communication  |
+
+#### Outbound Connections (Required)
+
+The relay server requires outbound connectivity to:
+
+| Protocol | Destination                          | Port | Purpose                                    |
+| -------- | ------------------------------------ | ---- | ------------------------------------------ |
+| TCP      | app.infisical.com / eu.infisical.com | 443  | API communication and certificate requests |
+
+## Firewall Configuration
+
+### Relay Firewall Rules
```

Comment:
we just need to tell them what needs to be allowed, etc which we already talk about. no need to go though each firewall